### DISCUSSION TITLE
deep merge default values?

### DISCUSSION THREAD

**Discussion by jasonkuhrt** [react-hook-form/react-hook-form#6942] [Q&A]
I was looking for this feature, then saw https://github.com/react-hook-form/react-hook-form/issues/5003, but in the current API there is no such feature nor is it mentioned in the website docs.

What has changed since then? Anyone know what's up?

**Comment by jasonkuhrt** [comment]
Perhaps part of the answer is that we're supposed to use https://react-hook-form.com/api/useform/reset.

**Comment by bluebill1049** [comment]
we don't do deep merge with `defaultValues` anymore.
